swagger_server/controllers/security_controller_.py

```diff
@@ -1,3 +1,3 @@
-
+from typing import List
```

Comment:
Is this import needed at all?

Reply:
This file was generated so I don't know if I should touch it